docs/changelog.rst

```diff
@@ -0,0 +1,2 @@
+.. _changelog:
+.. include:: ../CHANGELOG.rst
```

Comment:
@indirectlylit 
Do you think it necessary for the Release notes to be on the first level of the TOC (in the `index.rst`)?
How about we include it at the end of the [install.rst](https://github.com/learningequality/kolibri/edit/release-v0.7.x/docs/install.rst)? That way it's viewable once the reader opens the install topic, but is not included in the TOC.

I'm mostly saying this because it's a "techie" habit to check the release notes, most of the users don't usually read those before updating, and I tried to keep only the straightforward tasks in the TOC in the left sidebar...